evalai/urls.py

```diff
@@ -22,24 +22,31 @@
 from django.views.generic.base import TemplateView
 from drf_yasg import openapi
 from drf_yasg.views import get_schema_view
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView,
+)
+# from drf_spectacular.openapi import OpenApiInfo
 from rest_framework import permissions
 from rest_framework_expiring_authtoken.views import obtain_expiring_auth_token
 from web import views
 
+
 handler404 = "web.views.page_not_found"
 handler500 = "web.views.internal_server_error"
 
-swagger_api_info = openapi.Info(
-    title="EvalAI API",
-    default_version="v1",
-    description="EvalAI Documentation",
-    contact=openapi.Contact(email="team@cloudcv.org"),
-    license=openapi.License(name="BSD License"),
-)
+# swagger_api_info = OpenApiInfo(
```

Comment:
Please don't comment it, let's remove it since you are adding a different package.